packages/back-end/src/scim/users/patchUser.ts

```diff
@@ -0,0 +1,108 @@
+import { cloneDeep } from "lodash";
+import { Response } from "express";
+import { updateOrganization } from "../../models/OrganizationModel";
+import { getUserById } from "../../services/users";
+import { ScimPatchRequest, ScimUser } from "../../../types/scim";
+import { OrganizationInterface } from "../../../types/organization";
+
+async function removeUserFromOrg(
+  org: OrganizationInterface,
+  userIndex: number
+) {
+  const updatedOrg = cloneDeep(org);
+
+  // TODO: When we introduce the ability to manage roles via SCIM, we can remove this check.
+  const userIsAdmin = org.members[userIndex].role === "admin";
+
+  if (userIsAdmin) {
+    const numberOfAdmins = org.members.filter(
+      (member) => member.role === "admin"
+    );
+
+    if (numberOfAdmins.length === 1) {
+      throw new Error("Cannot remove the only admin");
+    }
+  }
+
+  updatedOrg.members.splice(userIndex, 1);
+
+  await updateOrganization(org.id, updatedOrg);
```

Comment:
We should only pass in the specific fields that are being updated, not the entire Organization object